### ISSUE TITLE
let's remove CHECK_APP_SIZE_OPTIONALLY related workaround

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/appium#238]
./files/mcloud/node_modules/@appium/base-driver/build/lib/basedriver/mcloud-utils.js

we implemented possibility to disable file size check functionality. the problem was due to the parallel calls by carina to the appcenter, s3 etc and blocking requests by provider.

in new 1.x carina generation we send a single request for the 1st thread only and wait.
So we can try to remove this workaround completely...
 
```
2023-09-04 20:08:47:402 - [BaseDriver] [BaseDriver] [MCLOUD] Local version of app was found. Will check actuality of the file
2023-09-04 20:08:47:405 - [BaseDriver] [BaseDriver] [MCLOUD] env CHECK_APP_SIZE_OPTIONALLY=undefined
2023-09-04 20:08:47:406 - [debug] [BaseDriver] [BaseDriver] [MCLOUD] Making GET http call for retrieving of remote app size
2023-09-04 20:08:47:514 - [debug] [BaseDriver] [BaseDriver] [MCLOUD] CONTENT-LENGTH for the file: 105147400
2023-09-04 20:08:47:516 - [BaseDriver] [BaseDriver] [MCLOUD] Remote file size is 105147400 and local file size is 105147400
2023-09-04 20:08:47:516 - [BaseDriver] [BaseDriver] [MCLOUD] Sizes are the same. Hence will use already stored application for the session
```

**Comment by vdelendik** [comment]
in scope of 2.0.4 we made it obligatory. if all good with problematic appcenter we can definitely remove this variable and appropriate workaround from our patched code.

**Comment by vdelendik** [comment]
merged, also #296 submitted to organize change including diff files. changes might be tested in 2.0.9-beta7

**Comment by dhreben** [comment]
Basic acceptance  with iOS-Preferences-Test ,Mobile-iOS-Safari-Test ,  Android-Demo-Test  successfully completed. Video, screenshots and logs are available
